crates/biome_deserialize/src/json.rs

```diff
@@ -138,6 +138,92 @@ impl DeserializableValue for AnyJsonValue {
     }
 }
 
+#[cfg(feature = "serde")]
+impl Deserializable for serde_json::Value {
+    fn deserialize(
+        value: &impl DeserializableValue,
+        name: &str,
+        diagnostics: &mut Vec<DeserializationDiagnostic>,
+    ) -> Option<Self> {
+        struct Visitor;
+        impl DeserializationVisitor for Visitor {
+            type Output = serde_json::Value;
+            const EXPECTED_TYPE: VisitableType = VisitableType::ARRAY;
```

Comment:
All types are visitable here:
```suggestion
            const EXPECTED_TYPE: VisitableType = VisitableType::all();
```